README.md

```diff
@@ -84,13 +84,15 @@ type to `pip-compile` to use this plugin for the respective environment.
 
 ### Configuration Options
 
-| name                   | type        | description                                                                                                                           |
-| ---------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------- |
-| lock-filename          | `str`       | The filename of the ultimate lockfile. `default` env is `requirements.txt`, non-default is `requirements/requirements-{env_name}.txt` |
-| pip-compile-constraint | `str`       | An environment to use as a constraint file, ensuring that all shared dependencies are pinned to the same versions.                    |
-| pip-compile-hashes     | `bool`      | Whether to generate hashes in the lockfile. Defaults to `false`.                                                                      |
-| pip-compile-verbose    | `bool`      | Set to `true` to run `pip-compile` in verbose mode instead of quiet mode, set to `false` to silence warnings                          |
-| pip-compile-args       | `list[str]` | Additional command-line arguments to pass to `pip-compile`                                                                            |
+| name                     | type        | description                                                                                                                           |
+| ------------------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| lock-filename            | `str`       | The filename of the ultimate lockfile. `default` env is `requirements.txt`, non-default is `requirements/requirements-{env_name}.txt` |
+| pip-compile-constraint   | `str`       | An environment to use as a constraint file, ensuring that all shared dependencies are pinned to the same versions.                    |
+| pip-compile-hashes       | `bool`      | Whether to generate hashes in the lockfile. Defaults to `false`.                                                                      |
+| pip-compile-verbose      | `bool`      | Set to `true` to run `pip-compile` in verbose mode instead of quiet mode, set to `false` to silence warnings                          |
+| pip-compile-installer    | `str`       | Whether to use `pip` or `pip-sync` to install dependencies into the project. Defaults to `pip`                                        |
+| pip-compile-args         | `list[str]` | Additional command-line arguments to pass to `pip-compile`                                                                            |
+| pip-compile-install-args | `list[str]` | Additional command-line arguments to pass to `pip-compile-installer`                                                                  |
```

Comment:
I'd suggest separating lock args from install args. At the very least in terms of order, but maybe also an actual separate group.

* ...
* pip-compile-verbose
* pip-compile-args
------------
* pip-compile-installer
* pip-compile-install-args